changelog.d/+9da8d07a.misc.md

```diff
@@ -0,0 +1 @@
+Removed support for Python 3.8 -- by {user}`sirosen`.
```

Comment:
This is definitely not a `misc` note. It should be [`breaking`](https://github.com/jazzband/pip-tools/blob/edd379ab2e24c91ce5e62dbb9b0f3ced4808137f/towncrier.toml#L39C18-L39C26)